lib/inv_ca_cg.cpp

```diff
@@ -53,6 +51,153 @@ namespace quda {
     if (!param.is_preconditioner) profile.TPSTOP(QUDA_PROFILE_FREE);
   }
 
+  CACGNE::CACGNE(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
+    CACG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(matSloppy.Expose()),
+    xp(nullptr), yp(nullptr), init(false) {
+  }
+
+  CACGNE::~CACGNE() {
+    if ( init ) {
+      if (xp) delete xp;
+      if (yp) delete yp;
+      init = false;
+    }
+  }
+
+  // CACGNE: M Mdag y = b is solved; x = Mdag y is returned as solution.
+  void CACGNE::operator()(ColorSpinorField &x, ColorSpinorField &b) {
```

Comment:
clang-format suggested the following change:


```
-  void CACGNE::operator()(ColorSpinorField &x, ColorSpinorField &b) {
+  void CACGNE::operator()(ColorSpinorField& x, ColorSpinorField& b)
+  {
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_cg.cpp -->